chalice/app.py

```diff
@@ -430,7 +433,8 @@ class Chalice(object):
 
     FORMAT_STRING = '%(name)s - %(levelname)s - %(message)s'
 
-    def __init__(self, app_name, debug=False, configure_logs=True, env=None):
```

Comment:
I would also rather not expose this through the initializer. Could we instead do something like we do for default binary content types where we expose some property off of the `Chalice` object. For example we could do something like:

```python
app = Chalice(...)
app.default_response_serializer = SuperGreatSerializerClass
```